src/couch/src/couch_debug.erl

```diff
@@ -326,6 +328,17 @@ help(print_table) ->
           - TableSpec: List of either {Value} or {Width, Align, Value}
             where Align is either left/center/right.
 
+        ---
+    ", []);
+help(print_report) ->
+    io:format("
+        print_report(Report)
+        --------------------------------
+
+        Print a report in table form.
+          - Report: List of {InfoKey, InfoVal} where each InfoKey is unique
+          (unlike print_table/2).
+
```

Comment:
In addition to the description of the Report format. Could you provide an example which illustrates the shape of the data expected by this function?

Reply:
As in the format the table prints in?

Reply:
yes